lib/flipper/views/layout.erb

```diff
@@ -39,18 +39,19 @@
       <% end %>
 
       <div class="vads-l-row">
-        <nav aria-label="breadcrumb" aria-live="polite" class="vads-l-col va-nav-breadcrumbs">
-          <ul class="va-nav-breadcrumbs-list vads-u-margin--0 vads-u-padding--0">
-            <% @breadcrumbs.each do |breadcrumb| %>
-              <li class="breadcrumb-item <% if breadcrumb.active? %>active<% end %>">
-                <% if breadcrumb.active? %>
-                  <%= breadcrumb.text %>
-                <% else %>
-                  <a href="<%= breadcrumb.href %>"><%= breadcrumb.text %></a>
-                <% end %>
-              </li>
+        <nav>
+          <ul class="nav nav-underline mb-3 align-items-center">
+            <% Flipper::UI.configuration.nav_items.each do |item| %>
+              <ul class="vads-u-margin--0 vads-u-padding--0 nav-item">
```

Comment:
We might want to update the ul class according to this [PR](https://github.com/flippercloud/flipper/commit/966a956f36cd0ef6a838b9ae6b20818c5509766c).
Here's the layout file:https://github.com/flippercloud/flipper/blob/966a956f36cd0ef6a838b9ae6b20818c5509766c/lib/flipper/ui/views/layout.erb. Here's what flipper suggests:
```
      <nav>
        <ul class="nav nav-underline mb-3 align-items-center">
          <% if Flipper::UI.configuration.application_href %>
            <li class="nav-item">
              <a class="nav-link" href="<%= Flipper::UI.configuration.application_href %>">
                ‹ App
              </a>
            </li>
          <% end %>


          <% Flipper::UI.configuration.nav_items.each do |item| %>
            <li class="nav-item">
              <a href="<%= url_for(item[:href]) %>" class="nav-link<%= " active" if request.url.start_with?(url_for(item[:href])) %>">
                <%= item[:title] %>
              </a>
            </li>
          <% end %>

          <li class="nav-item ms-auto text-muted small">
            <a href="https://www.flippercloud.io/docs/ui?utm_source=oss&utm_medium=ui&utm_campaign=docs">Docs</a>
            &bull;
            Version: <%= Flipper::VERSION %>
          </li>
        </ul>
      </nav>
```

Reply:
That's what I started with, but I changed it up just a little because I didn't love the way it looked 😆

Notable changes:
- I removed the link to Settings
- I made it an unordered list
- I changed the Docs link to link to our internal docs
- I removed the Version